spec/presenters/pull_requests_by_gem_spec.rb

```diff
@@ -122,5 +146,49 @@
         }
       ])
     end
+
+    it 'orders the gems by name' do
+      uglifier_pull_request = {
+        application_name: 'publisher',
+        title: 'Bump uglifier from 4.5.6 to 7.8.9',
+        url: 'https://www.github.com/alphagov/publisher/pull/456',
+        opened_at: Date.parse('2018-01-01 08:00:00')
+      }
+
+      gds_api_adapters_pull_request = {
+        application_name: 'frontend',
+        title: 'Bump gds-api-adapters from 1.2.3 to 4.5.6',
+        url: 'https://www.github.com/alphagov/frontend/pull/123',
+        opened_at: Date.parse('2018-01-01 08:00:00')
+      }
+
+      gds_api_adapters_pull_request2 = {
+        application_name: 'publisher',
+        title: 'Bump gds-api-adapters from 4.5.6 to 7.8.9',
+        url: 'https://www.github.com/alphagov/frontend/pull/456',
+        opened_at: Date.parse('2018-01-01 08:00:00')
+      }
+
+      result = described_class.new.execute(
+        [
+          uglifier_pull_request,
+          gds_api_adapters_pull_request,
+          gds_api_adapters_pull_request2
+        ]
+      )
+
+      expect(result).to eq([
+        {
+          gem_name: 'gds-api-adapters',
+          pull_requests: [
+            gds_api_adapters_pull_request,
+            gds_api_adapters_pull_request2
```

Comment:
The ordering here matches the ordering passed into `execute`.  Is that intentional?

Reply:
Good catch, updated.